src/adapters/model_mixin.py

```diff
@@ -40,22 +41,6 @@ def init_adapters(self, model_config, adapters_config, **kwargs):
         if hasattr(super(), "init_adapters"):
             super().init_adapters(self.config, self.adapters_config, **kwargs)
 
-        self.hook_after_embeddings(self._hook_fn)
-
-    def _hook_fn(self, module, args, output):
-        new_output = self.invertible_adapters_forward(output)
-        return new_output
-
-    def hook_after_embeddings(self, hook_fn: Callable):
```

Comment:
Why are we deleting/refactoring this?

Reply:
We now have 2 adapter methods that are hooked in after the embeddings (invertible adapter & prompt tuning) and have therefore decided to put them in the new `post_embedding_forward` function of the `ModelBaseAdaptersMixin`. This makes the implementation cleaner and it is easier to add additional adapter methods later, which need to be added after the embedding layer as well.